server/server.go

```diff
@@ -1,113 +1,52 @@
 package server
 
-import (
-	"path/filepath"
-
-	pb "github.com/kubernetes/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
-	"github.com/opencontainers/ocitools/generate"
-	"golang.org/x/net/context"
+const (
+	runtimeAPIVersion = "v1alpha1"
```

Comment:
@runcom , I have these questions because I kind of thought `RuntimeApiVersion` is specific to each container runtime after I read `api.pb.go`.

    type VersionResponse struct {
        // The version of the kubelet runtime API.
        Version *string `protobuf:"bytes,1,opt,name=version" json:"version,omitempty"`
        // The name of the container runtime.
        RuntimeName *string `protobuf:"bytes,2,opt,name=runtime_name" json:"runtime_name,omitempty"`
        // The version of the container runtime.
        RuntimeVersion *string `protobuf:"bytes,3,opt,name=runtime_version" json:"runtime_version,omitempty"`
        // The API version of the container runtime.
        RuntimeApiVersion *string `protobuf:"bytes,4,opt,name=runtime_api_version" json:"runtime_api_version,omitempty"`
        XXX_unrecognized  []byte  `json:"-"`
    }